.github/workflows/cd.yml

```diff
@@ -111,15 +109,19 @@ jobs:
     needs: push
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - name: Create distribution
-        run: python setup.py sdist bdist_wheel
-      - name: Publish to PyPI
-        uses: pypa/gh-action-pypi-publish@master
-        with:
-          user: __token__
-          password: ${{ secrets.pypi_password }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

Comment:
We have to create the `PYPI_USERNAME` secret though

Reply:
Actually, it can be set to `__token__`
https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives

Reply:
or that